tests/users/test_dao.py

```diff
@@ -148,6 +150,149 @@ def test_dao_delete_only_user_admin(self):
             (messages.USER_CANT_DELETE, HTTPStatus.BAD_REQUEST), dao_result
         )
 
+    def test_dao_delete_user(self):
+        dao = UserDAO()
+
+        user = UserModel(
+            name=user2["name"],
+            email=user2["email"],
+            username=user2["username"],
+            password=user2["password"],
+            terms_and_conditions_checked=user2["terms_and_conditions_checked"],
+        )
+        db.session.add(user)
+        db.session.commit()
+
+        # Verify that user was inserted in database through DAO
+        before_delete_user = UserModel.query.filter_by(id=2).first()
+        self.assertIsNotNone(before_delete_user)
```

Comment:
The comment above is self explanatory. Either remove the comment or make this change
```suggestion
        user = UserModel.query.filter_by(id=2).first()
        self.assertIsNotNone(user)
```